### PR TITLE
Added user login and new user login pages

### PR DESCRIPTION
**Pull request by drewlearnsabout**
Hi all, made some simple pages that could be used for the login page and new user login page.  Let me know your thoughts!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                        <label id="email-label" class="labels">Email</label>
+                        <input id="email" type="email" placeholder="Email" required>
+                    </section>
+                    <section id="admin_permissions_section">            
+                        <label id="dropdown-label">Admin permissions</label>
+                        <select id="dropdown" name="admin_permissions" class="dropdown">
+                            <option disabled value>Please select an option</option>
+                            <option value="yes">Yes</option>
+                            <option value="no">No</option>
+                            <option value="Default to no">Default to no</option>
```
Comment (NickSchimek) [inline index.html]: Default to no should not be an option. When the form displays the "no" value should be the default value.

**Comment by NickSchimek** [comment]
Great work! I like it. There is just one change that needs to be done for now.

We'll need to hold off on merging this until we get the front end structure built. Then we'll figure out where this needs to go, what name to give the files, etc...

The indentation seems off in a couple of places, but that's not really a big deal. I will try to get a style robot added so that it can automatically inform us of these little issues so that they're easily caught and fixed.

oh yeah, it seems there are duplicate files `index.html` & `index_new_user.html` is there a reason for both of files?

File: index_new_user.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                        <label id="email-label" class="labels">Email</label>
+                        <input id="email" type="email" placeholder="Email" required>
+                    </section>
+                    <section id="admin_permissions_section">            
+                        <label id="dropdown-label">Admin permissions</label>
+                        <select id="dropdown" name="admin_permissions" class="dropdown">
+                            <option disabled value>Please select an option</option>
+                            <option value="yes">Yes</option>
+                            <option value="no">No</option>
+                            <option value="Default to no">Default to no</option>
```
Comment (NickSchimek) [inline index_new_user.html]: Default to no should not be an option. When the form displays the "no" value should be the default value.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <main id="main">
+            <form id="new_user_form" action="/submit-info-here">
+                <section id="personal_information">
+                    <p id="personal_information_label">New User Login</p>
+                    <section id="first_name_section">    
+                        <label id="first-name-label" class="labels">First Name</label>
+                        <input id="first-name" type="text" placeholder="First Name" required>
```
Comment (NickSchimek) [inline index.html]: These validations are great work. Good job!

File: index.html (hunk 1 above)
Comment (drewlearnsabout) [reply index.html]: Hey Nick, sorry about the super late response, I'll work on it this week!

Comment (NickSchimek) [reply index.html]: No worries! That is perfect

**Comment by NickSchimek** [comment]
Thanks so much for this PR @drewlearnsabout - I'm looking forward to your react PR's. I'm going to go ahead and close this since @hmarcks has a design for us to convert to react.

**Pull request closed without merging**